src/applications/personalization/profile360/components/ContactInformation.jsx

```diff
@@ -19,23 +18,14 @@ import WorkPhone from '../vet360/components/WorkPhone';
 import FaxNumber from '../vet360/components/FaxNumber';
 import Email from '../vet360/components/Email';
 
+import { handleDowntimeForSection } from './DowntimeBanner';
+import ContactInformationExplanation from './ContactInformationExplanation';
+
 export default class ContactInformation extends React.Component {
 
   renderContent = () => {
-    if (every(Object.keys(this.props.user.profile.vet360), false)) {
-      return <LoadFail information="contact"/>;
-    }
```

Comment:
A newly initialized user will probably have all falsy properties, so this doesn't belong here anymore.